packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -307,7 +307,9 @@ function build(syscall, forVatID, cacheSize, vatPowers, vatParameters) {
     const p = makeImportedPromise(resultVPID);
 
     lsdebug(
-      `ls.qm send(${JSON.stringify(targetSlot)}, ${prop}) -> ${resultVPID}`,
+      `ls.qm send(${JSON.stringify(targetSlot)}, ${String(
+        prop,
+      )}) -> ${resultVPID}`,
```

Comment:
Line breaks inside template strings give me the willies.

Reply:
> Am I understanding that right?

Perfectly

> Line breaks inside template strings give me the willies.

The computer did it. Blame the computer.

Reply:
That's not what you'll say when the robots come for you.